Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift

```diff
@@ -384,19 +384,25 @@ struct MacroSystem {
 // MARK: - MacroApplication
 
 /// Removes attributes from a syntax tree while maintaining their surrounding trivia.
-private class AttributeRemover: SyntaxRewriter {
+@_spi(AttributeRemover)
```

Comment:
I would just name this `@_spi(Testing)` because it describes the intent that it’s only exposed publicly for testing.

Reply:
Done.